jxl_test_utils/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+#[macro_export]
```

Comment:
`pub use assert_almost_eq` works if staying in the same crate.

Reply:
move to util/test.rs